### ISSUE TITLE
`--cursor` flag doesn't work in grimblast

### ISSUE THREAD

**Issue by ItsProfessional** [hyprwm/contrib#92]
### Program

grimblast

### Maintainers of the program

@fufexan, @Misterio77

### Bug or Regression?

Bug

### Description

There's no cursor in the screenshot (nor in the preview when selecting the area, nor in the final output) even if the `--cursor` flag is passed.

### How to reproduce

- Run `grimblast --freeze --cursor copy area`
- Paste the copied screenshot on any application
- There's no cursor to be seen

**Comment by fufexan** [comment]
Ah, you're using `--cursor` with `area`. Which obviously doesn't work, as the cursor would be outside the area. Or be in the top-left corner, which looks very weird.

**Comment by faergeek** [comment]
It felt weird to me that there's no separate tool for freezing the screen (or I just couldn't find one). I haven't tried grimblast, I've tried hyprshot, not sure who came up with that first, but both use hyprpicker to freeze the screen. It's kinda hacky and in both cases it looks complicated and only worked less than half of the time for me for whatever reason.

Instead of figuring out why it didn't work for me I've just made a simple tool focused just on "freezing" the screen: https://github.com/faergeek/still

It takes a screenshot from every output and overlays it via a layer-shell, letting slurp and grim or whatever else do their thing. Can optionally include the cursor on the screenshot if you want it.

I have no idea if it works properly with multiple monitors, I only have one at the moment and can't test it. But otherwise it works perfectly for me.

Integrating it might be problematic though, it only exists in AUR and exists at all for less than 24 hours.

**Comment by faergeek** [comment]
Yes, makes sense to open an issue in "still" repo, let's figure it out there.

**Comment by faergeek** [comment]
v0.0.2 should probably fix this

**Comment by faergeek** [comment]
Got a chance to test with multiple monitors. v0.0.2 still behaves weirdly since it calls a passed command for every output. Fixed it in v0.0.3. Terminal on the right in the screenshot is also on a monitor with a different scaling. Left one is 1.5, right one is 2.0. Seems to work fine, except bluriness on the left, which I believe is not caused by `still`, as it doesn't really do anything with pixels it gets via screencopy protocol at all and just passes them along to a layer shell.

On the left it also shows that the cursor can be overlayed on top.

My bind for it in hyprland config looks like this:

```
bind = $mainMod, S, exec, still -p -c 'slurp -d | grim -g- -l0 -' -p | swappy -f -
```

<img width="2500" height="720" alt="Image" src="https://github.com/user-attachments/assets/f87a066e-4442-481a-836e-b907ce3b1b7c" />

**Comment by fufexan** [comment]
> Left one is 1.5, right one is 2.0. Seems to work fine, except bluriness on the left, which I believe is not caused by `still`

Correct, that's grim upscaling the lower scale monitor to match the higher scale one. Glad to see this working.
